From: Alex Henrie <alexhenrie24@gmail.com>
Subject: [PATCH 1/4] ntdll: Add missing break to case IPV6_PKTINFO
Message-Id: <20210806052648.422314-1-alexhenrie24@gmail.com>
Date: Thu,  5 Aug 2021 23:26:25 -0600

Signed-off-by: Alex Henrie <alexhenrie24@gmail.com>
---
It looks like I lost the break statement in rebasing, sorry.
---
 dlls/ntdll/unix/socket.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/ntdll/unix/socket.c b/dlls/ntdll/unix/socket.c
index 4f1b9a468eb..35c846eb7ae 100644
--- a/dlls/ntdll/unix/socket.c
+++ b/dlls/ntdll/unix/socket.c
@@ -458,6 +458,7 @@ static int convert_control_headers(struct msghdr *hdr, WSABUF *control)
                         ptr = fill_control_message( WS_IPPROTO_IPV6, WS_IPV6_PKTINFO, ptr, &ctlsize,
                                                     (void *)&data_win, sizeof(data_win) );
                         if (!ptr) goto error;
+                        break;
                     }
 #endif /* IPV6_PKTINFO */
 

-- 
2.32.0

